accepted/2021/statics-in-interfaces/README.md

```diff
@@ -934,36 +1029,10 @@ namespace System
 
         // The following methods are approved but not yet implemented in the libraries
 
-        static abstract TSelf AcosPi(TSelf x);
-
-        static abstract TSelf AsinPi(TSelf x);
-
-        static abstract TSelf AtanPi(TSelf x);
-
-        static abstract TSelf Atan2Pi(TSelf y, TSelf x);
-
         static abstract TSelf Compound(TSelf x, TSelf n);
 
-        static abstract TSelf CosPi(TSelf x);
-
-        static abstract TSelf ExpM1(TSelf x);
-
-        static abstract TSelf Exp2(TSelf x);
-
-        static abstract TSelf Exp2M1(TSelf x);
-
-        static abstract TSelf Exp10(TSelf x);
-
-        static abstract TSelf Exp10M1(TSelf x);
-
         static abstract TSelf Hypot(TSelf x, TSelf y);
```

Comment:
Hypot is likewise interesting, but the most logical place would be in `IRootFunctions`